cmd/crowdsec/main.go

```diff
@@ -357,18 +357,9 @@ func main() {
 		return
 	}
 
-	/*TBD : need to be cleaned up a bit*/
-	/*if the user is in "single file mode" (might be writting scenario or parsers),
-	allow loading **without** parsers or scenarios */
-	// if cConfig.SingleFile == "" {
-	// 	if len(parsers.nodes) == 0 {
-	// 		log.Fatalf("no parser(s) loaded, abort.")
-	// 	}
-
-	// 	if len(holders) == 0 {
-	// 		log.Fatalf("no bucket(s) loaded, abort.")
-	// 	}
-	// }
+	if cConfig.API == nil || cConfig.API.Client == nil || cConfig.API.Client.Credentials == nil {
+		log.Fatalf("Missing Client API credentials, abort.")
```

Comment:
We need to be more specific on which config is missing (local or online API credentials)